Codeinwp/themeisle-companion
# Fix msp icon placement

### Summary
Fixes issue where editor meta box icon was out of place.


### Will affect visual aspect of the product

NO

### Screenshots 
<img width="285" alt="Screenshot at Jan 25 13-46-19" src="https://user-images.githubusercontent.com/9929553/105701934-ba53bf00-5f13-11eb-8a76-3a5e7e05310c.png">
<img width="300" alt="Screenshot at Jan 25 13-44-29" src="https://user-images.githubusercontent.com/9929553/105701769-7c569b00-5f13-11eb-9629-133dff02974a.png">


### Test instructions
- add a new post or edit an existing one
- open the top right sidebar from the editor the Neve icon should be there
- it should be properly placed
